MatrixSDK/JSONModels/MXJSONModels.h

```diff
@@ -1125,6 +1125,25 @@ FOUNDATION_EXPORT NSString *const kMXPushRuleScopeStringGlobal;
 
 @end
 
+@interface MXKeysQueryResponseRaw : MXJSONModel
+
+    /**
+     The device keys per devices per users.
+     */
+    @property (nonatomic) NSDictionary<NSString *, id> *deviceKeys;
```

Comment:
In the existing MXKeysQueryResponse this was a `MXUsersDevicesMap<MXDeviceInfo*> *deviceKeys`